README.md

```diff
@@ -17,22 +17,20 @@ Local, and Qubes executors are available.
 Fedora:
 
 ```bash
-$ sudo dnf install python3-packaging python3-click python3-lxml createrepo_c devscripts gpg python3-pyyaml rpm docker python3-docker podman python3-podman reprepro python3-pathspec rpm-sign rb_libtorrent-examples openssl tree mock python3-jinja2-cli pacman m4 asciidoc rsync sequoia-sq sequoia-sqv sequoia-chameleon-gnupg
+$ sudo dnf install $(cat dependencies-fedora.txt)
+$ test "${QUBES_ENV_SOURCED:-}" = "1" || sudo dnf install qubes-gpg-split
```

Comment:
Why conditional on this variable? If you try to detect if running in qubes, then we have [`/usr/share/qubes/marker-vm`](https://www.qubes-os.org/faq/#what-is-the-canonical-way-to-detect-qubes-vm)